protocol-tests/src/main/java/fixtures/dpgcustomization/implementation/models/LroProduct.java

```diff
@@ -2,14 +2,14 @@
 // Licensed under the MIT License.
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
-package fixtures.dpgcustomization.models;
+package fixtures.dpgcustomization.implementation.models;
 
 import com.azure.core.annotation.Fluent;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-/** The LROProduct model. */
+/** The LroProduct model. */
 @Fluent
-public final class LROProduct extends Product {
+public final class LroProduct extends Product {
```

Comment:
Here a some example of the output.

Previously I just use HLC flow to generate these and put to tests folder.
Now I switched to DPG with `generate-models` to generate these in main folder (but in implementation.models).